### DISCUSSION TITLE
Allow globbing inside of boolean conditionals

### DISCUSSION THREAD

**Discussion by jriddle-sf** [community/community#31507] [Other Feature Feedback, Questions, & Ideas] [answered]
My organization migrated from a self-hosted Github installation to Github.com. To prevent collisions, we applied prefixes to our project names for the team/service who owns it.

**Example Repository Names:**

```
PREFIX.PROJECT_NAME_00
PREFIX.PROJECT_NAME_01
PREFIX.PROJECT_NAME_02
```

A nice search feature would be to exclude anything with that prefix from my search.

**Query Example:**

```
org:MY_TARGET_ORG NOT repo:MY_TARGET_ORG/PREFIX.*
```

Perhaps this can be done but I'm not writing the query correctly. I've tried escaping the `.` and more with no success.

**Comment by jriddle-sf** [comment, marked as answer]
Well... I just reviewed the syntax page and found this.

> Note: We do not currently support regular expressions or partial matching for repository names, so you’ll have to type the entire repo name (including the user prefix) for the repo:... qualifier to work

> **Source:** https://cs.github.com/about/syntax

My apologies.

**Reply by colinwm** [reply]
We are working on supporting that, though! Stay tuned 😄
